repositories.yaml

```diff
@@ -379,6 +393,7 @@ applications:
       - org.mozilla.appservices:logins
       - org.mozilla.components:support-migration
       - org.mozilla.components:places
+      - org.mozilla.components:fxa-client
```

Comment:
Did Fenix actually start using it yet?
I see https://github.com/mozilla-mobile/firefox-android/pull/3918 but that's closed and defered.

Reply:
I'm not sure if they are using it yet or not, all I know is that the FxA folks have been testing this via the Debug View and were wondering why it hadn't shown up in the Dictionary yet.  It looks like this upstream PR needs to land first: https://github.com/mozilla/application-services/pull/5887

Reply:
As long as the metrics.yaml location won't change I don't mind landing this now already

Reply:
I think we're using it.  Here's the [metrics.yaml file](https://github.com/mozilla/application-services/blob/main/components/fxa-client/android/metrics.yaml) and the [code that uses it](https://github.com/mozilla/application-services/blob/f3cd74035a5eafcdd5925aadc2859ec59583ecdb/components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/PersistedFirefoxAccount.kt#L500-L518).

I think last issue is probe-scaper, although I definitely could be missing something.

Reply:
Thanks! Somehow my search didn't turn that up.